### PR TITLE
Improve documentation for event sample

### PR DESCRIPTION
**Pull request by majakurcius**
Changes proposed in this pull request:

- Improve documentation for event sample

**Related issue(s)**
See also #5690

### CODE REVIEW

File: docs/application-connector/08-05-trigger-lambda-with-event.md [hunk 1]
```diff
@@ -64,35 +75,35 @@ This guide shows how to create a simple lambda function and trigger it with an e
    }
    ```
 
-2. Get the `externalName` of the Service Class of the registered service.
+3. Expose the `externalName` of the Service Class of the registered service.
```
Comment (kazydek) [inline docs/application-connector/08-05-trigger-lambda-with-event.md]:
```suggestion
3. Expose `externalName` of the Service Class of the registered service.
```

File: docs/application-connector/08-05-trigger-lambda-with-event.md [hunk 2]
```diff
@@ -3,16 +3,26 @@ title: Trigger a lambda with events
 type: Tutorials
 ---
 
-This guide shows how to create a simple lambda function and trigger it with an event.
+To create a simple lambda function and trigger it with an event, you must first register a service using the Application Registry in Application Connector. This service then sends the event that triggers the lambda. You must create a Service Instance which enables this event in the Namespace. Follow this guide to learn how to do it.
```
[the remark this answers is not in the document]
Comment (kazydek) [reply docs/application-connector/08-05-trigger-lambda-with-event.md]: Could you capitalize Event in the whole doc?

File: docs/application-connector/08-05-trigger-lambda-with-event.md (hunk 1 above)
Comment (majakurcius) [reply docs/application-connector/08-05-trigger-lambda-with-event.md]: I'd leave it as it is as in this particular case, the `externalName` doesn't just function as a variable name, but is also an integral part of the sentence.

File: docs/application-connector/08-05-trigger-lambda-with-event.md (hunk 2 above)
[the remark this answers is not in the document]
Comment (majakurcius) [reply docs/application-connector/08-05-trigger-lambda-with-event.md]: I'm not sure we should capitalize it as it does not refer to the Kubernetes resource. 

Good point with the `Application Connector` part!

File: docs/application-connector/08-05-trigger-lambda-with-event.md [hunk 3]
```diff
@@ -3,16 +3,26 @@ title: Trigger a lambda with events
...
 
-1. Register a service with the following specification to the desired App.
+1. Export the name of the Namespace to which you bound your Application, and the name of your Application.
+
+   ```bash
+   export NAMESPACE={YOUR_NAMESPACE}
+   export APP_NAME={YOUR_APPLICATION_NAME}
+   ```
+
+2. Register a service with events with this example AsyncAPI specification to the desired Application.
```
[the remark this answers is not in the document]
Comment (majakurcius) [reply docs/application-connector/08-05-trigger-lambda-with-event.md]: Yes, it is. Thanks!

**Pull request merged**